x-pack/plugins/security_solution/common/endpoint/types/file_storage.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * The Metadata information about a file that was uploaded by Endpoint
+ * as a result of a `get-file` response action
+ */
+export interface UploadedFile {
```

Comment:
FYI - having had some more time looking at the file upload and how we will use it in the UI, I'm not sure that we will need this Type. Or we might needed it, but not under `common/` - maybe on the server side only. We might not actually need to interact with the file stored because the Endpoint action response will likely have everything we need.